src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
```

Comment:
this won't work for all 0.6-dev versions, should be more specific about version-depedent cutoffs or use `isdefined`